slides/Navigation/source/SLAM.html

```diff
@@ -38,177 +38,140 @@
 
 * The goal of SLAM  is to create or enhance an environment map.
 
+* Mathematically,
+    - Given the Robot's Controls and Observations
+    - Calculate the Map of the environment and Path of the robot.
+
 * Maps represent the environment, either 2D or 3D.
 
-* Is actually a tough problem to solve!
+* Is actually a tough problem to solve! Troubling humanity since 1985 [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam01-intro.pdf)
 
 ---
 
-## Mapping - SLAM
-### Algorithms
-
-* Filtering:
-  * Particle filter (Gmapping eloquent), EKF(extended Kalman filter)
-  * Model the problem as Online state estimation
+## Mapping - Algorithms
 
-* Smoothing:
-  * Full trajectories are estimated using a set of measurements
-  * Graph-based SLAM, e.g. Pose graph optimization (Cartographer, SLAM toolbox, Karto SLAM)
+* Kalman Filters - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam08-kf-wrapup.pdf)
+    - Probabilistic, Bayes filter, best for Linear Models and Gaussian distributions
+    - Further specialisations - Extended KF, Unscented KF, Extended Information Filter (EIF), Sparse efficiency
+* Particle Filters - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam09-particle-filter.pdf)
+    - Probabilistic, Recursive Bayes filter, also appropriate for non-Gaussian distributions
+    - AKA Monte Carlo Localisation - most commonly used
+* Graph based - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam17-ls-landmarks.pdf)
+    - Deterministic, Graph of a problem with nodes for poses & edges for spatial constraints between poses
+    - Build the graph and find a node configuration that minimizes the error introduced by the constraints
 
```

Comment:
Shall we mention some packages here?

Reply:
Its a good idea, but we would need to dig into each package to see what type of algorithm they use, and I just ran out of time to do it. If you are able to do so, please go ahead!